cmd/k8srules/controller.go

```diff
@@ -0,0 +1,273 @@
+// Copyright 2016 IBM Corporation
+//
```

Comment:
Same comment as above. If we add a command line flag to the controller that it is running as a K8S service, then it could automatically switch to using this code instead of the generic code.